views/view_package.dt

```diff
@@ -177,6 +177,10 @@ block body
 			dt Score:
 			dd#score(title="#{formatPackageStats(stats)}")!= formatScore(req.rootDir, stats.score)
 
+			dt Short URL:
+				dd
+					a(href="http://#{packageName}.dub.pm", target="_blank") #{packageName}.dub.pm
```

Comment:
Both http://mir.dub.pm and http://dub.pm/p/mir work - I thought sub-domains are easier.

In case someone worries about SSL - see https://letsencrypt.org/2017/07/06/wildcard-certificates-coming-jan-2018.html (tl;dr; this should be possible soon)